cobra/README.md

```diff
@@ -40,9 +40,9 @@ cobra init --pkg-name github.com/spf13/newApp path/to/newApp
 Once an application is initialized, Cobra can create additional commands for you.
 Let's say you created an app and you wanted the following commands for it:
 
-* app serve
-* app config
-* app config create
+- app serve
```

Comment:
Please, do not mix syntax changes in the same commit. Create another commit if you want to change the lists, admonitions, etc.